### PR TITLE
Remove parent/pom.xml for Event Hubs Track 1

### PR DESCRIPTION
**Pull request by conniey**

### CODE REVIEW

**Comment by mitchdenny** [comment]
@conniey looks like you'll need to remove the intermediate parent POM reference from the other packages.

File: pom.data.xml [hunk 1]
```diff
@@ -543,7 +543,9 @@
   <modules>
     <module>./sdk/batch/microsoft-azure-batch</module>
     <module>./sdk/cosmos</module>
```
Comment (conniey) [inline pom.data.xml]: @mitchdenny Do I need to bump the version of pom.data.xml because I updated the modules?

**Pull request merged**